src/sandbox/snapshotSandbox.ts

```diff
@@ -8,9 +8,10 @@ import { SandBoxType } from '../interfaces';
 function iter(obj: typeof window, callbackFn: (prop: any) => void) {
   // eslint-disable-next-line guard-for-in, no-restricted-syntax
   for (const prop in obj) {
-    if (obj.hasOwnProperty(prop)) {
-      callbackFn(prop);
-    }
+    // ie10,11下默认hasOwnProperty不包括clearInterval等
+    // if (obj.hasOwnProperty(prop)) {
```

Comment:
Why need change this method? It is consistent both in active or inactive period, while iterator can not catch clearInterval property on window.

Reply:
run this  `window.hasOwnProperty('clearInterval')`   under  ie10,11.  Then you can see  'false',but  'true'   under chrome.   It could lead to  that snapshotSandbox can not revert under  ie10,11.

Reply:
@kuitos  it  is different in inactive and in active,because   you rewrite  cleartInterval   in  interval.ts   `global.clearInterval=` in active. Once you rewrite clearInterval,`window.hasOwnProperty('clearInterval')` will be `true` .  This causes differences between in inactive and in active.